### DISCUSSION TITLE
flowise integration in next.js project

### DISCUSSION THREAD

**Discussion by rithasyed** [FlowiseAI/Flowise#2201] [Q&A]
can i integrate flowise in my next.js project by just installing and importing? if so how can i do it ? or is there another way for integration ?

**Comment by toi500** [comment]
Check this

https://github.com/FlowiseAI/FlowiseEmbedReact

**Comment by ksaigowthamraju** [comment]
I have also same question not integrating the chatbot can i integrate the flowise project into the next js project with our changes
